### PR TITLE
OSDOCS-11723-13: Documented known issue on vSphere for OVNK plugin

### PR DESCRIPTION
**Pull request by dfitzmau**
**CM:**
CM sent out on the 5th of September. See the email named "[Notice of change] Networking known issue in 4.13 when upgrading your cluster".

**Version(s):**
4.13 only

**Issue:**
[OSDOCS-11723](https://issues.redhat.com/browse/OSDOCS-11723)

**Link to docs preview:**
* [About the OpenShift SDN network plugin](https://80633--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/openshift_sdn/about-openshift-sdn.html)
* [About the OVN-Kubernetes network plugin](https://80633--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/about-ovn-kubernetes)
* [4.13 Release Notes KNown Issues](https://80633--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes#ocp-4-13-known-issues)

- [x] SME has approved this change. (Ben Bennett)
- [x] QE has approved this change. (Ross Brattain)

Additional information:
* https://issues.redhat.com/browse/OCPSTRAT-1369
* https://docs.openshift.com/container-platform/4.16/installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned-network-customizations.html#nw-operator-vsphere-multiple-subnets_installing-vsphere-installer-provisioned-network-customizations
* For more information about setting the parameter, see xref:../../installing-vsphere-installer-provisioned-network-customizations.adoc#nw-operator-vsphere-multiple-subnets_installing-vsphere-installer-provisioned-network-customizations[Specifying multiple subnets for your network] (Installing on vSphere).

### CODE REVIEW

File: networking/openshift_sdn/rollback-to-ovn-kubernetes.adoc [hunk 1]
```diff
@@ -8,6 +8,11 @@ toc::[]
 
 As a cluster administrator, you can rollback to the OVN-Kubernetes network plugin from the OpenShift SDN network plugin if the migration to OpenShift SDN is unsuccessful.
 
+[IMPORTANT]
+====
+On {vmw-full}, if you upgraded your {product-title} 4.12 cluster to {product-title} {product-version} and you want your upgraded cluster to use the OVN-Kubernetes network plugin, you might experience a node VIP address issue with the Cloud Controller Manager. This issue impacts the ability of the Cloud Controller Manager to pull `quay.io` images into your cluster. As a workaround for this issue, specify the shared VIP address in the `networkSubnetCidr` parameter of your cluster configuration.
+====
+
```
Comment (rbbratta) [inline networking/openshift_sdn/rollback-to-ovn-kubernetes.adoc:15]: The `failed to get node address from cloud provider that matches ip` error is also seen on OpenShiftSDN, so I don't think we need a note for rollback.

**Review by lpettyjo — COMMENTED** [review]
LGTM!

**Pull request merged**